src/tests/auth/find.test.js

```diff
@@ -0,0 +1,56 @@
+import chai from 'chai';
```

Comment:
better you rename it to `<find-user.test.js>` because you are finding a user. `<find.test.js>` is not descriptive